### ISSUE TITLE
model.eval() in local_analysis.py

### ISSUE THREAD

**Issue by jxllp** [cfchen-duke/ProtoPNet#12]
First of all, thank you for sharing the source code of this inspiring work. It's really appreciated!

After training the model, the results I was getting making predictions with local_analysis.py didn’t match the accuracy calculated by the method test() in train_and_test.py.

Please correct if I am wrong, but I believe it’s because in local_analysis.py, model.eval() is missing after loading the model.
